api/client.go

```diff
@@ -49,7 +49,7 @@ type Client struct {
 	Compliance      *ComplianceService
 	Events          *EventsService
 	Integrations    *IntegrationsService
-	LQL             *LQLService
+	Query           *QueryService
```

Comment:
If Query is using V2 endpoints, could we move it to the V2 service that groups them all?

Reply:
@afiune is it cool that the entire service isn't currently, but will be soon using all v2 endpoints ?